README.md

```diff
@@ -1,2 +1,11 @@
 # CoC
 Code of Conduct of the Risk of Rain 2 modding community
+
+General server rules
+0: Follow Discord/GitHub ToS for general rules regarding what is and is not acceptable.
+1: Don't be an asshole. This includes, but is not limited to, harrasment,  death threats, generally calling people shitty names, racism, sexism, ect.
+2: What Moderators/Admins/Owners say is final. Period. They get to say what does and doesn't happen on the server. If you believe someone's abusing their power, bring it up with me.
+3: There is no rule #3
+4: NSFW content is not allowed. Period. No appeal.
+5: Try to keep content to the appropriate channels. General channel can be used for anything as long as you're not interrupting someone else.
+6: Don't beg for your mod idea to be created! Chances are, most of the mod devs have their own ideas. Maybe try to learn yourself, we're trying to make mod creation easy.
```

Comment:
I'd instead make a reference to the ideas channel for this, since that is a place for all kinds of ideas.